README.md

```diff
@@ -182,7 +182,7 @@ router.use(
 
 The `mapToUserId` is a customization point you need to override to set the value of the _fusebit.userId_ tag for the identity that is established in the authorization process (in the example above, to Slack). This value would be typically derived from the authentication mechanism you are using to protect the endpoint above. For example, it could come from a cookie-based session.
 
-The `mapToTenantId` is used set the value of the _fusebit.tenantId_ tag for the new identity. This value could be implied by your application model and authentication context. For example, the specific authenticated user of your app may be part of a given company or organization who is the tenant of you app. If you don't provide an explicit value for the _fusebit.tenantId_ tag, its value will be set to the same value you provided for _fusebit.userId_.
+The `mapToTenantId` is used to set the value of the _fusebit.tenantId_ tag for the new identity. Your application model and authentication context could imply this value. For example, the specific authenticated user of your app may be part of a given company or organization who is the tenant of your app. If you don't provide an explicit value for the _fusebit.tenantId_ tag, its value will be the same value you provided for _fusebit.userId_.
```

Comment:
> Your application model and authentication context could imply this value.

I don't understand what this sentence means, so it might need a refactor.